### DISCUSSION TITLE
[ FF7 ] Crash - Disc 1

### DISCUSSION THREAD

**Discussion by alexis-guillosson** [julianxhokaxhiu/FFNx#218] [Q&A]
[FFNx.log](https://github.com/julianxhokaxhiu/FFNx/files/6135437/FFNx.log)

[crash.zip](https://github.com/julianxhokaxhiu/FFNx/files/6135438/crash.zip)

Crash when trying to return to midgar just after entering world map for the first time

**Comment by julianxhokaxhiu** [comment]
@alexis-guillosson Please update your FFNx installation to the latest canary or update the pack you're using ( which comes with the latest FFNx tested version ).
